web/Services/PropertyService.cs

```diff
@@ -68,13 +72,15 @@ public async Task<int> AddNewProperty(int landlordId, PropertyViewModel createMo
             AcceptsWithoutGuarantor = createModel.LandlordRequirements.AcceptsWithoutGuarantor,
 
             Rent = createModel.Rent,
-
-            Availability = createModel.Availability ?? AvailabilityState.Draft,
+            
             TotalUnits = createModel.TotalUnits ?? 1,
             OccupiedUnits = createModel.OccupiedUnits ?? 0,
-            AvailableFrom = createModel.Availability == AvailabilityState.AvailableSoon
-                ? createModel.AvailableFrom
-                : null
+            
+            Availability = createdAvailability,
+            AvailableFrom = createdAvailability == AvailabilityState.Available
+            ? createModel.Availability == AvailabilityState.AvailableSoon

```

Comment:
indenatation has gone weird here

Reply:
The indentation is probably weird because of the nested ternary - I'm not even sure how this should be indented to begin with, I though it looked right the way it currently is.